Sj small fixes

stephjuneau opened the pull request:
Small fixes to the Readme file and tutorial (including using "Image" to display the GIF as suggested by Robert).

stephjuneau committed:
Add files via upload

Uploaded new version of slides after small fixes (SPARCL contents table and adding GitHub link to tutorial)

stephjuneau committed:
fixed GIF display

stephjuneau committed:
Update README.md

small fixes to readme (on slides and adding the Rendered Notebook in the list)

stephjuneau committed:
small fixes to tutorial

stephjuneau committed:
updated RENDERED nb and date

stephjuneau commented:
@rnikutta, I re-ran the RENDERED notebook after DL services were back online and changed to today's date for both notebooks. Could you take a look and merge? Then we can copy them over for the other Tutorial for this week.

rnikutta reviewed (changes requested):
Hi @stephjuneau , I've reviewed README.md and RENDERED_...ipnb

README.md

- Fix the When and Where at the beginning (it still shows the info for ADASSx)
- Once moved to a new repo, adjust also the instructions which tutorial repo to clone (it current says "ADASSx")
- Remove this line" `Tutorial info: https://pretalx.com/adassx-2025/talk/YRPKFE/`

RENDERED NB

- "redshift slides of interest include a galaxy group identified by DESI (Williams et al., in prep.)" --> Is it still in prep?
- I woudl change the name of the link to the Data Explorer in this sentence "Note: The column names and descriptions can be found directly in a notebook with qc.schema() or from the [Data Lab Query Interface](https://datalab.noirlab.edu/data-explorer) or using the Table Access Protocol (TAP) service with a compatible tool such as TOPCAT." from "Data Lab Query Interface" to "Data Lab's Data Explorer".
- The 2nd code cell under "Query to SDSS DR17" begins with a spurious commented-out line `# sdss_dr17.specobj                          # SDSS DR17 specobj table`. Remove it.If it's meant as an explanation, the line should probably be moved to the comment block below the query.
- Save for the first code cell under "Query DESI DR1 for the same region"
- In this code line ` is_central = (abs(result_sdss['plug_ra']-ra_central)<1./3600)&(abs(result_sdss['plug_dec']-dec_central)<1./3600)` add white spaces around the & operator, as per PEP-8). Similarly for the | operator in `is_sdss = (df_spec['data_release']=='SDSS-DR16')|(df_spec['data_release']=='BOSS-DR16')`, and in few other cells.
- The figure produced at the end of the section "Joint query to SDSS DR17 & LS DR10" could use a legend. Unclear what is shown.

stephjuneau commented:
Hi @rnikutta , the new repo is here: https://github.com/astro-datalab/Tutorial-DESI-NOIRLab-2025 with an updated README file. I will address your other comments in both places for the notebook!

stephjuneau committed:
fixes from Robert review

stephjuneau commented:
Hi @rnikutta ,

Thanks for the review! See replies in-line below:

> Hi @stephjuneau , I've reviewed README.md and RENDERED_...ipnb
> 
> README.md
> 
> * Fix the When and Where at the beginning (it still shows the info for ADASSx)
> * Once moved to a new repo, adjust also the instructions which tutorial repo to clone (it current says "ADASSx")
> * Remove this line" `Tutorial info: https://pretalx.com/adassx-2025/talk/YRPKFE/`

Addressed in new repo: https://github.com/astro-datalab/Tutorial-DESI-NOIRLab-2025 

> RENDERED NB
> 
> * "redshift slides of interest include a galaxy group identified by DESI (Williams et al., in prep.)" --> Is it still in prep?

Yes, still in prep.

> * I woudl change the name of the link to the Data Explorer in this sentence "Note: The column names and descriptions can be found directly in a notebook with qc.schema() or from the [Data Lab Query Interface](https://datalab.noirlab.edu/data-explorer) or using the Table Access Protocol (TAP) service with a compatible tool such as TOPCAT." from "Data Lab Query Interface" to "Data Lab's Data Explorer".

Done in both notebooks (LSS and RENDERED)

> * The 2nd code cell under "Query to SDSS DR17" begins with a spurious commented-out line `# sdss_dr17.specobj                          # SDSS DR17 specobj table`. Remove it.If it's meant as an explanation, the line should probably be moved to the comment block below the query.

Moved it below as an explanation in both notebooks

> * Save for the first code cell under "Query DESI DR1 for the same region"

Moved it below as an explanation in both notebooks

> * In this code line ` is_central = (abs(result_sdss['plug_ra']-ra_central)<1./3600)&(abs(result_sdss['plug_dec']-dec_central)<1./3600)` add white spaces around the & operator, as per PEP-8). Similarly for the | operator in `is_sdss = (df_spec['data_release']=='SDSS-DR16')|(df_spec['data_release']=='BOSS-DR16')`, and in few other cells.

Fixed this in 3 cells in both notebooks.

> * The figure produced at the end of the section "Joint query to SDSS DR17 & LS DR10" could use a legend. Unclear what is shown.

Fixed on both panels in both notebooks.

stephjuneau committed:
adjusted GIF for rendered to display in GitHub
